src/graphql/Offer.graphql

```diff
@@ -48,6 +48,9 @@ query Offer($id: ID) {
           currency
         }
       }
+      ... on NoDiscount {
```

Comment:
I think it's sufficient to check `__typename`? 🙂

Reply:
You're right, I just removed this now 👍